src/engines/html5/media-source/adapters/native-adapter.js

```diff
@@ -544,7 +544,8 @@ export default class NativeAdapter extends BaseMediaSourceAdapter {
           language: '',
           height: videoHeight,
           width: videoWidth,
-          active: true
+          active: true,
+          index: 0
```

Comment:
this means we will always say the index is 0 even if track changes

Reply:
fixed